provision/initramfs/capabilities/provision-vnfs/70-kernelmodules

```diff
@@ -33,4 +33,8 @@ if [ -x "${NEWROOT}${DEPMOD}" ]; then
     chroot "${NEWROOT}" /sbin/depmod -a
 fi
 
+if [ -e "${NEWROOT}"/lib/modules/"${KVERSION}"/sysctl.conf ]; then
+    ln -s /lib/modules/"${KVERSION}"/sysctl.conf" "${NEWROOT}"/boot/sysctl.conf-"${KVERSION}"
```

Comment:
Thinking we should check if `"${NEWROOT}"/boot/sysctl.conf-"${KVERSION}"` exists before creating this symlink.

Reply:
Indeed. Thanks for catching!
Let me see if I can push an updated patch. I will fix the changelog entry to better explain the rationale.